AsyncApp/Migrations/20200911235133_AddRoomAmenities.cs

```diff
@@ -0,0 +1,165 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace AsyncApp.Migrations
+{
+    public partial class AddRoomAmenities : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropTable(
+                name: "Amenities");
+
+            migrationBuilder.DeleteData(
+                table: "Room",
+                keyColumn: "Id",
+                keyValue: 1);
+
+            migrationBuilder.DeleteData(
+                table: "Room",
+                keyColumn: "Id",
+                keyValue: 2);
+
+            migrationBuilder.DeleteData(
+                table: "Room",
+                keyColumn: "Id",
+                keyValue: 3);
+
+            migrationBuilder.AlterColumn<long>(
+                name: "Id",
+                table: "Room",
+                nullable: false,
+                oldClrType: typeof(int),
+                oldType: "int")
+                .Annotation("SqlServer:Identity", "1, 1")
+                .OldAnnotation("SqlServer:Identity", "1, 1");
+
```

Comment:
Don't forget to update the `Down()` with an equivalent/opposite change!

```suggestion
            migrationBuild.DropPrimaryKey("PK_Room", "Room");

            migrationBuilder.AlterColumn<long>(
                name: "Id",
                table: "Room",
                nullable: false,
                oldClrType: typeof(int),
                oldType: "int")
                .Annotation("SqlServer:Identity", "1, 1")
                .OldAnnotation("SqlServer:Identity", "1, 1");

            migrationBuild.AddPrimaryKey("PK_Room", "Room", "Id");

```